Table-sorting-by-click/test.js

```diff
@@ -0,0 +1,38 @@
+const table = document.querySelector(".grid");
+
+function sortTable(colNum, type) {
+  const tbody = table.querySelector("tbody");
+
+  const arr = Array.from(tbody.rows);
+
+  let compare;
```

Comment:
why is it called "compare"?

Reply:
Next we compare rows to sort, would it be better to call it sorting?